rules/linux/credential_access_proc_credential_dumping.toml

```diff
@@ -0,0 +1,63 @@
+[metadata]
+creation_date = "2023/04/26"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/04/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of the mimipenguin exploit script which is linux adaptation of Widnows tool mimikatz.
+Mimipenguin exploit script is used to dump clear text passwords from a currently logged-in user. The tool exploits
+a known vulnerability CVE-2018-20781. Malicious actors can exploit the cleartext credentials in memory by
+dumping the process and extracting lines that have a high probability of containing cleartext passwords.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Linux Credential Dumping via Proc Filesystem"
+references = [
+    "https://github.com/huntergregal/mimipenguin",
+    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-20781"
+]
+risk_score = 47
+rule_id = "ef100a2e-ecd4-4f72-9d1e-2f779ff3c311"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Credential Access"]
+type = "eql"
+query = '''
+sequence by process.parent.name with maxspan=1m
```

Comment:
Maybe you can consider including more "sequence by" fields here to fine tune the sequence of events and eliminate unintentionally capturing these sequence events across multiple users or multiple hosts. 
So something like  `sequence by process.parent.name, host.name (or maybe host.ip or host.id), user.name`